backend/fms_core/migrations/0021_v3_3_0.py

```diff
@@ -36,68 +37,69 @@ def create_platforms_and_instrument_types(apps, schema_editor):
     InstrumentType = apps.get_model("fms_core", "InstrumentType")
 
     INSTRUMENT_TYPES_BY_PLATFORM = {
-        "LS454": ["454 GS",
-                  "454 GS 20",
-                  "454 GS FLX",
-                  "454 GS FLX+",
-                  "454 GS FLX Titanium",
-                  "454 GS Junior",
-                  ],
+        "LS454": [ "454 GS",
```

Comment:
What had to be changed?

Reply:
Only alignments ...

Reply:
I dont think we should add spaces after brackets before the elements.. if that's what leads to changing the alignment

Reply:
ok i'll revert those spaces